wordpress-mobile/WordPress-iOS
# [Jetpack Content Migration]  Display Migration UI when Content Migration Feature Flag is enabled

## Related PR
The following PR should be merged before this one:
- https://github.com/wordpress-mobile/WordPress-iOS/pull/19632

## Description
This PR adds the ability the display the Migration UI by enabling the "Content Migration" feature flag.

<img src="https://user-images.githubusercontent.com/9609223/203836693-6bd38f72-7326-4461-90e2-f8807d9a2034.png" width=300 />

_Sites hidden for privacy reasons_

## Test Instructions
1. Open Jetpack app.
2. Log into your account.
3. My Site tab > "Me" button > App Settings > Debug
4. Enable "Content Migration" Feature Flag
5. Close and re-open Jetpack app
6.  **Expect** the Migration Welcome screen to show up

## Regression Notes
1. Potential unintended areas of impact
None.

7. What I did to test those areas of impact (or what existing automated tests I relied on)
None.

8. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:
